lib/terse-class-list/test/terse-class-list.js

```diff
@@ -40,5 +40,5 @@
             });
         });
     });
-    
+
```

Comment:
I prefer the old name, _terse_ is more like _concise_ to me (ie high information density). The class names are just jibberish and carries no information, they are simply short. Also one might get the impression that the list itself is terse. Makes sense?